src/main/java/com/beerjournal/breweriana/item/rating/RatingController.java

```diff
@@ -0,0 +1,37 @@
+package com.beerjournal.breweriana.item.rating;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.validation.annotation.Validated;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.Collection;
+
+@RestController
+@RequestMapping("/api/ratings")
+@RequiredArgsConstructor
+public class RatingController {
+
+    private final RatingService ratingService;
+
+    @GetMapping("item/{itemId}")
```

Comment:
I haven't realized that `/api/ratings/user` corresponds to ratings created by user, not ratings about users as I thought. But neither mine nor yours api is correct :P I think `/api/ratings?itemId=id` suits most

PS Is `/api/ratings/item/{itemId}` shorter than `/api/item/{itemId}/ratings`? :P